src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,118 @@
+                    + "Input params: " + driver, e);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String getDriverRequest = "SELECT * FROM drivers WHERE id = ? AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getDriverStatement =
+                        connection.prepareStatement(getDriverRequest,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            getDriverStatement.setLong(1, id);
+            ResultSet resultSet = getDriverStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.ofNullable(createDriverObject(resultSet));
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get driver by id. ID = " + id, e);
+        }
+    }
+
+    @Override
+    public List<Driver> getAll() {
+        String getAllRequest = "SELECT * FROM drivers WHERE is_deleted = FALSE;";
+        List<Driver> allDrivers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllStatement.executeQuery(getAllRequest);
+            while (resultSet.next()) {
+                allDrivers.add(createDriverObject(resultSet));
+            }
+            return allDrivers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all drivers.", e);
+        }
+    }
+
+    @Override
+    public Driver update(Driver driver) {
+        String updateDriverRequest = "UPDATE drivers "
+                + "SET name = ?, license_number = ? "
+                + "WHERE id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateDriverStatement =
+                        connection.prepareStatement(updateDriverRequest)) {
+            updateDriverStatement.setString(1, driver.getName());
+            updateDriverStatement.setString(2, driver.getLicenseNumber());
+            updateDriverStatement.setLong(3, driver.getId());
+            updateDriverStatement.executeUpdate();
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update data for driver: " + driver, e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteDriverRequest = "UPDATE drivers "
+                + "SET is_deleted = TRUE "
+                + "WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteDriverStatement =
+                        connection.prepareStatement(deleteDriverRequest)) {
+            deleteDriverStatement.setLong(1, id);
+            return deleteDriverStatement.executeUpdate() > 1;
```

Comment:
```suggestion
            return deleteDriverStatement.executeUpdate() > 0;
```
if you change single row in the table that means method worked as expected

Reply:
done